### DISCUSSION TITLE
Geodetic system?

### DISCUSSION THREAD

**Discussion by amalone-eog** [jannikmi/timezonefinder#244] [Q&A] [answered]
First off - thank you for writing this library. It's perfect for quick timezone determination of remote/mobile IoT devices.

My question:
**What geodetic system is this library using for latitude/longitude comparisons against the timezone shape files?**

**Comment by jannikmi** [comment]
Thank you for the feedback and starting the discussion. This package uses the timezone boundary data from here: https://github.com/evansiroky/timezone-boundary-builder/releases

I believe information about the geodetic system in use can be found there.

**Reply by amalone-eog** [reply]
Thanks for the response - I reached out to the folks at timezone-boundary-builder and got an answer for us.

**Comment by amalone-eog** [comment, marked as answer]
For anyone else with the same question - TimezoneFinder adheres to the geodetic system used by timezone-boundary-builder, which uses WGS84 as its geodetic system. See link below for the discussion on timezone-boundary-builder's repo.

[https://github.com/evansiroky/timezone-boundary-builder/issues/180#issuecomment-2349327643](https://github.com/evansiroky/timezone-boundary-builder/issues/180#issuecomment-2349327643)
